lessons/2_more_strings_and_functions/lesson.py

```diff
@@ -134,6 +152,15 @@ def my_cool_function(pet_name):
 
 # # ANSWER D GOES HERE
 
+def is_number_positive(number):
+    if number == 0:
+        return "Neither"
+    else:
+        return number >= 1
+
```

Comment:
This code is good but there are some things that can be improved here. 
The first thing is that this if statement doesn't need an else. If number is not equal to zero, the next line of code is going to run anyway. So that part can be simplified to this:

```
def is_number_positive(number):
    if number == 0:
        return "Neither"
    return number >=1
```
There is one more thing though, and I think this was covered later in the lesson. Python objects have truthiness values and numbers that aren't zero in Python3 are "true" and zero is "false" So any number that is true is going to be positive or negative. False will be zero. Since the number itself is truthy, we don't need to create a boolean by checking number == 0. 

```
def is_number_positive(number):
    if not number:
        return "Neither"
    return number >= 1
```